quinn-proto/src/connection/streams.rs

```diff
@@ -21,6 +21,7 @@ use crate::{
     Dir, Side, StreamId, TransportError, VarInt, MAX_STREAM_COUNT,
 };
 
+#[allow(missing_docs)]
```

Comment:
Rather than spamming this annotation across every fuzzing-only interface, could we replace the crate-level `#![warn(missing_docs)]` with `#![cfg_attr(not(fuzzing), warn(missing_docs))]` or similar?